.github/workflows/build-artifact.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Build cf.gov artifact'
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
```

Comment:
Thinking a little about the overall workflow here, would we not want to build on PRs as well?

If this is replacing our Jenkins deployable-zipfile builder, and we want to deploy work-in-progress to dev servers, then we'll need to get an artifact from somewhere. Why not just always be building?

Reply:
Yeah, I think I was a bit too parsimonious here. I was thinking that in those cases we could just dispatch the workflow, but I guess there isn't really much cost associated with always building.

Reply:
Done in 816cdd239. I think it will actually be quite nice and feel decently magical to be able to deploy any pushed commit.

Reply:
(The build will of course fail until https://github.com/cfpb/consumerfinance.gov/pull/7732 is merged)